### ISSUE TITLE
Addon locale is not working. All tooltip strings are shown as english. My WOW era client is koKR.

### ISSUE THREAD

**Issue by KwangMyungKim** [Lethay/TalentedClassic#121]
Description of each talent is shown as english. Default button strings are shown as korean.

It looks talent tooltip locale data is included in talented_data. 

But it's not applied well.

Is there something to do?

**Comment by INSPRT** [comment]
This is true. I am also using the "koKR" client, and Locale is not working.
